### DISCUSSION TITLE
Add chia genesis message to the services starting screen in GUI for a bit of nostalgia, ascii art quote displayed in CLI

### DISCUSSION THREAD

**Discussion by OverActiveBladderSystem** [Chia-Network/chia-blockchain-gui#2370] [Ideas]
Add the genesis challenge message (from Bram), for the mainnet launch, into the GUI services starting screen for a bit of history and mystery to all who launch the program, just for those couple seconds it takes before transitioning to the main screen. (for CLI users a little ascii art quote of the message before the daemon starts?)

I think it would be a nice touch and homage for users to see it for that brief moment.

`"Operational error causes Fed payment system to crash" We stand on the shoulders of giants, now let's get farming!`

**Comment by BrandtH22** [comment]
@OverActiveBladderSystem , note I transferred this enhancement request to the GUI repo for better visibility 👍
